### PR TITLE
Bump Kanela to 1.0.11

### PR DESCRIPTION
**Pull request by dpsoft**
* Java `16` support
* Fix invalid bytecode in exception handler(https://github.com/kamon-io/Kamon/issues/1008)
* Revert `suppress = classOf[Throwable]` in `Redis` and `JDBC` moudules
* Minor Cleanup

### CODE REVIEW

File: instrumentation/kamon-redis/src/main/resources/reference.conf [hunk 1]
```diff
@@ -11,4 +11,4 @@ kanela.modules {
       "redis.clients.jedis.Protocol",
     ]
   }
-}
+}
```
Comment (SimunKaracic) [inline instrumentation/kamon-redis/src/main/resources/reference.conf:14]: Gimme back that newline! :D

**Review by SimunKaracic — COMMENTED** [review]
There's also a kanela agent version in build.sbt:
https://github.com/kamon-io/Kamon/blob/36226d098cb6daa8f330ebccb0aa16ea35c456dc/build.sbt#L684

Update that too please, or if you feel like it dig around and try to figure out why it's declared twice.

Other than that, great PR, thanks for the contribution! :tada:

File: instrumentation/kamon-redis/src/main/resources/reference.conf (hunk 1 above)
Comment (dpsoft) [reply instrumentation/kamon-redis/src/main/resources/reference.conf]: done! :)

**Comment by dpsoft** [comment]
@SimunKaracic please don't merge. The play tests fail.

**Comment by dpsoft** [comment]
@SimunKaracic  this is the error: `[main] ERROR 2021-06-02 09:44:19  Logger : Error => scala.util.Failure with message Illegal parameter name of final scala.Function0 default for public java.lang.Object scala.util.Failure.getOrElse(scala.Function0). Class loader: sun.misc.Launcher$AppClassLoader@18b4aac2: java.lang.IllegalStateException: Illegal parameter name of final scala.Function0 default for public java.lang.Object scala.util.Failure.getOrElse(scala.Function0)`

**Comment by SimunKaracic** [comment]
I'm not sure what to do with that, do you think it's caused by something other than bumping kanela to the new version?

**Comment by dpsoft** [comment]
I found the issue, I will fix and make a new minor kanela release.

**Comment by dpsoft** [comment]
@SimunKaracic done!

**Comment by SimunKaracic** [comment]
Great job @dpsoft!

**Pull request merged**